train_management/views/dayplanning.py

```diff
@@ -89,6 +89,27 @@ def get_success_url(self):
         return reverse_lazy("day-planning-detail", kwargs={'pk': self.object.id})
 
 
+@method_decorator(login_required, name='dispatch')
+class DayPlanningRecipientView(generic.DetailView):
+    model = DayPlanning
+    form_class = DayPlanningFieldsetForm
+    template_name_suffix = "_recipient_form"
+
+    def get_context_data(self, **kwargs):
+        copy_recipients = CopyRecipient.objects.all()
+        all_active_personnel = _get_personnel()
+        working_personnel_train = Personnel.objects.filter(functionpersons__train__in=self.object.train_set.all())
+        working_personnel_stationary = Personnel.objects.filter(functionpersons__dayplanning=self.object)
+        all_working_personnel = working_personnel_train | working_personnel_stationary
+        all_working_personnel_distinct = all_working_personnel.distinct()
+
+        return super().get_context_data(
```

Comment:
Ist das so üblich in django? Verstehe noch nicht ganz, warum hier der Supercall notwendig ist.